InvocationTargetException with "psiPackage" is null message in console log

Caused by: java.lang.NullPointerException: Cannot invoke "com.intellij.psi.PsiPackage.getAnnotations()" because "psiPackage" is null

This can occur when the package name is invalid.

Similar to #132 but it also happens on Mac.

--- a/src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/annotations/AnnotationDiagnosticsCollector.java
+++ b/src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/annotations/AnnotationDiagnosticsCollector.java
@@ -68,10 +68,12 @@ public class AnnotationDiagnosticsCollector extends AbstractDiagnosticsCollector
 
             PsiPackage psiPackage = JavaPsiFacade.getInstance(unit.getProject())
                     .findPackage(unit.getPackageName());
-            PsiAnnotation[] pkgAnnotations = psiPackage.getAnnotations();
-            for (PsiAnnotation annotation : pkgAnnotations) {
-                if (isValidAnnotation(annotation.getQualifiedName(), validAnnotations))
-                    annotatables.add(new Tuple.Two<>(annotation, psiPackage));
+            if (psiPackage != null) {
+                PsiAnnotation[] pkgAnnotations = psiPackage.getAnnotations();
+                for (PsiAnnotation annotation : pkgAnnotations) {
+                    if (isValidAnnotation(annotation.getQualifiedName(), validAnnotations))
+                        annotatables.add(new Tuple.Two<>(annotation, psiPackage));
+                }
             }
 
             PsiClass[] types = unit.getClasses();

